Nice, thanks. Looks like it is good (except the extra print statement in your changes maybe). Below is current loghub release raising an exception, then update to your branch, then rerun with expected output.

```
$ loghub pytroll/pycoast -t "$LOGHUB_GITHUB_TOKEN" -st v1.3.2 -plg bug "Bugs fixed" -plg enhancement "Features added" -plg documentation "Documentation changes"

LOGHUB: Querying issues since tag "v1.3.2"

Traceback (most recent call last):
  File "/home/davidh/miniconda3/envs/loghub_test/bin/loghub", line 10, in <module>
    sys.exit(main())
  File "/home/davidh/miniconda3/envs/loghub_test/lib/python3.7/site-packages/loghub/cli/main.py", line 52, in main
    parse_arguments(skip=False)
  File "/home/davidh/miniconda3/envs/loghub_test/lib/python3.7/site-packages/loghub/cli/main.py", line 299, in parse_arguments
    show_related_issues=options.show_related_issues,
  File "/home/davidh/miniconda3/envs/loghub_test/lib/python3.7/site-packages/loghub/core/formatter.py", line 381, in create_changelog
    show_prs=show_prs)
  File "/home/davidh/miniconda3/envs/loghub_test/lib/python3.7/site-packages/loghub/core/formatter.py", line 453, in render_changelog
    show_prs=show_prs)
  File "/home/davidh/miniconda3/envs/loghub_test/lib/python3.7/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/home/davidh/miniconda3/envs/loghub_test/lib/python3.7/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/home/davidh/miniconda3/envs/loghub_test/lib/python3.7/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "<template>", line 25, in top-level template code
  File "/home/davidh/miniconda3/envs/loghub_test/lib/python3.7/site-packages/jinja2/environment.py", line 452, in getitem
    return obj[argument]
jinja2.exceptions.UndefinedError: 'pri' is undefined


$ pip install git+https://github.com/goanpeca/loghub.git@fix/errors
Collecting git+https://github.com/goanpeca/loghub.git@fix/errors
  Cloning https://github.com/goanpeca/loghub.git (to revision fix/errors) to ./pip-req-build-_7m21k1y
  Running command git clone -q https://github.com/goanpeca/loghub.git /tmp/pip-req-build-_7m21k1y
  Running command git checkout -b fix/errors --track origin/fix/errors
  Switched to a new branch 'fix/errors'
  Branch 'fix/errors' set up to track remote branch 'fix/errors' from 'origin'.
Requirement already satisfied: jinja2 in /home/davidh/miniconda3/envs/loghub_test/lib/python3.7/site-packages (from loghub==0.5.0.dev0) (2.11.2)
Requirement already satisfied: requests in /home/davidh/miniconda3/envs/loghub_test/lib/python3.7/site-packages (from loghub==0.5.0.dev0) (2.24.0)
Requirement already satisfied: MarkupSafe>=0.23 in /home/davidh/miniconda3/envs/loghub_test/lib/python3.7/site-packages (from jinja2->loghub==0.5.0.dev0) (1.1.1)
Requirement already satisfied: idna<3,>=2.5 in /home/davidh/miniconda3/envs/loghub_test/lib/python3.7/site-packages (from requests->loghub==0.5.0.dev0) (2.10)
Requirement already satisfied: certifi>=2017.4.17 in /home/davidh/miniconda3/envs/loghub_test/lib/python3.7/site-packages (from requests->loghub==0.5.0.dev0) (2020.6.20)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /home/davidh/miniconda3/envs/loghub_test/lib/python3.7/site-packages (from requests->loghub==0.5.0.dev0) (1.25.9)
Requirement already satisfied: chardet<4,>=3.0.2 in /home/davidh/miniconda3/envs/loghub_test/lib/python3.7/site-packages (from requests->loghub==0.5.0.dev0) (3.0.4)
Building wheels for collected packages: loghub
  Building wheel for loghub (setup.py) ... done
  Created wheel for loghub: filename=loghub-0.5.0.dev0-py2.py3-none-any.whl size=37837 sha256=90b093ac0f510e24f5caf81c6534cb6d5c884f57dae9bcccf54d2ae329541032
  Stored in directory: /tmp/pip-ephem-wheel-cache-bsngnvc7/wheels/7e/6d/34/218ccd7fa54d803675a563f34f2a39944c8553f2a7df23509e
Successfully built loghub
Installing collected packages: loghub
  Attempting uninstall: loghub
    Found existing installation: loghub 0.5.0
    Uninstalling loghub-0.5.0:
      Successfully uninstalled loghub-0.5.0
Successfully installed loghub-0.5.0.dev0


$ loghub pytroll/pycoast -t "$LOGHUB_GITHUB_TOKEN" -st v1.3.2 -plg bug "Bugs fixed" -plg enhancement "Features added" -plg documentation "Documentation changes"

LOGHUB: Querying issues since tag "v1.3.2"

/home/davidh/miniconda3/envs/loghub_test/lib/python3.7/site-packages/loghub/templates/changelog_pr_groups.txt
###############################################################################
## Version <RELEASE_VERSION> (2020/07/04)

### Issues Closed
...
```

--- a/loghub/templates/changelog_pr_groups.txt
+++ b/loghub/templates/changelog_pr_groups.txt
@@ -22,7 +22,7 @@ In this release {{ issues|length }} issue{% if issues|length != 1 %}s{% endif %}
 #### {{ label_group }}
 
 {%     for pr in grouped_prs -%}
-* [PR {{ pr['number'] }}](https://github.com/{{ repo_full_name }}/pull/{{ pr['number'] }}) - {{ pr['title'] }}, by [@{{ pri['user']['login'] }}]({{ pri['user']['html_url'] }})
+* [PR {{ pr['number'] }}](https://github.com/{{ repo_full_name }}/pull/{{ pr['number'] }}) - {{ pr['title'] }}
 {%-      if pr['loghub_related_issues']|length %} (
 {%-          for pri in pr['loghub_related_issues'] -%}
                  [{{ pri['text'] }}]({{ pri['url'] }})

